nerfstudio/engine/schedulers.py

```diff
@@ -107,3 +107,36 @@ def func(step):
 
         scheduler = lr_scheduler.LambdaLR(optimizer, lr_lambda=func)
         return scheduler
+
+
+@dataclass
+class NeuSSchedulerConfig(SchedulerConfig):
+    """Basic scheduler config with self-defined exponential decay schedule"""
+
+    _target: Type = field(default_factory=lambda: NeuSScheduler)
+    """target class to instantiate"""
+    warm_up_end: int = 5000
+    """Iteration number where warmp ends"""
+    learning_rate_alpha: float = 0.05
+    """Learning rate alpha value"""
+    max_steps: int = 300000
+    """The maximum number of steps."""
+
+
+class NeuSScheduler(Scheduler):
+    """Starts with a flat lr schedule until it reaches N epochs then applies a given scheduler"""
```

Comment:
Nit: this seems like a useful scheduler for other general methods, might be better to call it something like "DelayedScheduler" or something?

Reply:
Sure, I can change to that

Reply:
I'm not sure thats the best name, it turns out it is a Cosine decay scheduler. So perhaps `CosineDecayScheduler`. Here is a plot
![image](https://user-images.githubusercontent.com/3310961/222504973-3df470fe-4b08-44be-b24d-b8542b847236.png)

Reply:
I'll rename to CosineDecayScheduler 👍